_posts/2017-10-06-few-steps-to-fast-redshift.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Few Steps to Fast Redshift
+tags: Redshift performance
+comments: true
+hidden: true
+author: agnieszka
+---
+
+Do you think a web dashboard which communicates directly with Amazon Redshift and shows tabels, charts, numbers - statistics in general can do good? We believed it is, as long as the dashboard is used by a few users. As this was our case, we decided to give it a go. 
```

Comment:
> can do good? We believed it is 

It sounds strange :) Perhaps "can work well? We believed it could" or "can work well? We believe it can" would be better.

Reply:
I did sound strange to me as well, but couldn't think of anything better. Thanks :)